worlds/metroidprime/data/Tricks.py

```diff
@@ -0,0 +1,824 @@
+        TrickDifficulty.Easy,
+        lambda world, state: can_space_jump(world, state)
+        and can_bomb(world, state)
+        and can_power_beam(world, state)
+        and can_combat_ghosts(world, state),
+    )
+    hall_of_elders_reflecting_pool_no_spider = TrickInfo(
+        "Hall of Elders Reflecting Pool No Spider Ball",
+        "Reach the reflecting pool without the spider ball",
+        TrickDifficulty.Easy,
+        lambda world, state: can_space_jump(world, state)
+        and can_wave_beam(world, state)
+        and can_bomb(world, state)
+        and can_power_beam(world, state)
+        and can_combat_ghosts(world, state),
+    )
+    hall_of_elders_reflecting_pool_no_wave_beam = TrickInfo(
+        "Hall of Elders Reflecting Pool No Wave Beam",
+        "In Hall of the Elders, you can Hyper Bomb Jump (HBJ) to the morph ball track and reach the door to Reflecting Pool Access.",
+        TrickDifficulty.Medium,
+        lambda world, state: can_space_jump(world, state)
+        and can_bomb(world, state)
+        and can_power_beam(world, state)
+        and can_combat_ghosts(world, state),
+    )
+
+    hall_of_elders_elder_chamber_no_spider = TrickInfo(
+        "Hall of Elders Elder Chamber No Spider Ball",
+        "Reach the bomb slots without the spider ball by jumping on a peg to activate the top bomb slot",
+        TrickDifficulty.Easy,
+        lambda world, state: can_bomb(world, state)
+        and can_plasma_beam(world, state)
+        and can_space_jump(world, state)
+        and can_power_beam(world, state)
+        and can_combat_ghosts(world, state),
+    )
+    hall_of_elders_item_no_spider = TrickInfo(
+        "Hall of Elders Item No Spider Ball",
+        "Reach the bomb slots without the spider ball by jumping on a peg to activate the top bomb slot",
+        TrickDifficulty.Easy,
+        lambda world, state: can_bomb(world, state)
+        and can_ice_beam(world, state)
+        and can_space_jump(world, state)
+        and can_power_beam(world, state)
+        and can_combat_ghosts(world, state),
+    )
+
+    reflecting_pool_space_jump_climb = TrickInfo(
+        "Reflecting Pool Space Jump Climb",
+        "Climb the reflecting pool by space jumping off a stone toad",
+        TrickDifficulty.Easy,
+        can_space_jump,
+    )
+    reflecting_pool_nsj_climb: TrickInfo = TrickInfo(
+        "Reflecting Pool No Space Jump Climb",
+        "Perform an HBJ, UBJ, or BSJ off a stone toad",
+        TrickDifficulty.Hard,
+        can_bomb,
+    )
+
+    # Magmoor
+
+    lava_lake_item_suitless = TrickInfo(
+        "Lava Lake Item Suitless",
+        "Reach the Lava Lake item without the Varia Suit",
+        TrickDifficulty.Medium,
+        lambda world, state: can_missile(world, state)
+        and can_space_jump(world, state)
+        and has_energy_tanks(world, state, 4)
+        and state.can_reach(
+            "Magmoor Caverns: " + RoomName.Lake_Tunnel.value, None, world.player

```

Comment:
This rule is being used on entrances, but the `"Magmoor Caverns: Lake Tunnel"` region is not being registered as an indirect condition for the entrances. e.g. entrance `"Blue Door from Lava Lake to Pit Tunnel"`.

Reply:
Ah nice, I didn't know about this👍 I was able to register the indirect condition for both of these with minimal effort 🎉
(I changed the condition slightly for the lake tunnel one to use a room that actually makes a bit more sense, in case you notice that change as well)